rpcs3/Emu/RSX/D3D12/D3D12GSRender.cpp

```diff
@@ -720,12 +700,37 @@ bool D3D12GSRender::LoadProgram()
 	if (m_set_blend)
 	{
 		prop.Blend.RenderTarget[0].BlendEnable = true;
+
+		if (m_set_blend_mrt1)
+			prop.Blend.RenderTarget[1].BlendEnable = true;
+		if (m_set_blend_mrt2)
+			prop.Blend.RenderTarget[2].BlendEnable = true;
+		if (m_set_blend_mrt3)
+			prop.Blend.RenderTarget[3].BlendEnable = true;
```

Comment:
I'm noob but.. These seems duplicate. What if these `m_set_blend_mrt1` were stored into an array? These three should all be able to happen as said in your code.